src/brevitas_examples/imagenet_classification/ptq/ptq_evaluate.py

```diff
@@ -168,6 +168,47 @@
     help='Narrow range for weight quantization (default: enabled)')
 parser.add_argument(
     '--gpfq-p', default=0.25, type=float, help='P parameter for GPFQ (default: 0.25)')
+parser.add_argument(
+    '--quant_format',
+    default='int',
+    choices=['int', 'float'],
+    help='Quantization format to use for weights and activations (default: int)')
+parser.add_argument(
+    '--layerwise-first-last-mantissa-bit-width',
+    default=4,
+    type=int,
+    help='TODO'  # @TODO: write helpful description
```

Comment:
Help="Mantissa bit width used in case of float layerwise quantization for first and last layer"
Same for exponent